sdk/include/unwind.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
+#include <cdefs.h>
+#include <setjmp.h>
+
+/**
+ * On-stack linked list of cleanup handlers.
+ */
+struct CleanupList
+{
+	/// Next pointer.
+	CleanupList *next;
+	/// Jump buffer to return to.
+	__jmp_buf env;
+};
+
+/**
+ * Head of the cleanup list.
+ *
+ * This is stored in the space that the switcher reserves at the top of the
+ * stack.
```

Comment:
Add some commentary about this being guaranteed to be zero'd because it's managed by the switcher?  Otherwise it looks, from C, like the head is on stack and never initialized.